docs/hub/hugging-face-hub.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Hugging Face Hub
+---
+
+<h1>Hugging Face Hub documentation</h1>
+
+## What's the Hugging Face Hub?
+
+We are helping the community work together towards the goal of advancing Artificial Intelligence 🔥.
+
+Not one company, even the Tech Titans, will be able to “solve AI” by themselves – the only way we'll achieve this is by sharing knowledge and resources. We are building the largest open-source collection of models, datasets, and metrics on the Hugging Face Hub to democratize and advance AI for everyone 🚀. The Hugging Face Hub works as a central place where anyone can share and explore models and datasets.
+
+## What can you find on the Hub?
+
+The Hugging Face Hub hosts Git-based repositories, which are storage spaces that can contain all your files. 💾
+
+On it, you'll be able to upload and discover...
+
+* Models, *hosting the latest state-of-the-art models for NLP, vision, and audio tasks*
+* Datasets, *featuring a wide variety of data for different domains and modalities*
+* Spaces, *interactive apps for demonstrating ML models directly in your browser*
+
+Unlike other hosting solutions, the Hub offers **versioning, commit history, diffs, branches, and over a dozen library integrations**! You can learn more about the features that all repositories share in the **Repositories documentation**.
+
+## Models
+Models on the Hugging Face Hub allow for simple discovery and usage to maximize model impact. To promote responsible model usage and development, Model repos are equipped with [Model Cards](https://arxiv.org/abs/1810.03993) to inform users of each model's limitations and biases. Additional [metadata](/docs/hub/model-repos#model-card-metadata) about info such as their tasks, languages, and metrics can be included, with training metrics charts even added if the repository contains [TensorBoard traces](https://huggingface.co/models?filter=tensorboard). It's also easy to add an **inference widget** to your model, allowing anyone to play with the model directly in the browser! For production settings, an API is provided to **instantly serve your model**.
```

Comment:
```suggestion
Models on the Hugging Face Hub allow for simple discovery and usage to maximize model impact. To promote responsible model usage and development, model repos are equipped with [Model Cards](https://arxiv.org/abs/1810.03993) to inform users of each model's limitations and biases. Additional [metadata](/docs/hub/model-repos#model-card-metadata) about info such as their tasks, languages, and metrics can be included, with training metrics charts even added if the repository contains [TensorBoard traces](https://huggingface.co/models?filter=tensorboard). It's also easy to add an **inference widget** to your model, allowing anyone to play with the model directly in the browser! For production settings, an API is provided to **instantly serve your model**.
```

Reply:
For model card, instead of linking to the paper, we should likely link to our corresponding doc about them

Reply:
Agreed! I'll replace the link with a `TODO` for the moment, which I can adjust to point to our "Model Cards" doc once that's ready in a new PR 😄

Reply:
SG!